buildbot/master/files/config/factories.py

```diff
@@ -115,19 +115,17 @@ def make_step(self, command):
 ])
 
 
-def make_cd_string(command):
-    return "cd /c/buildbot/slave/windows/build ; " + command
+def make_cd_command(command):
+    return ["bash", "-l", "-c", "cd /c/buildbot/slave/windows/build ; " + command]
```

Comment:
You'll need to split this onto two lines to fix the lint error (see the Travis logs for details). Also, the space between `build` and the semicolon can be removed.

I'd suggest using one line to construct the cd-prefixed string, and the second line to return the whole array.